src/main/java/seedu/address/logic/commands/DeleteClaimCommand.java

```diff
@@ -87,4 +87,21 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(String.format(e.getMessage(), insuranceId, Messages.format(clientToEdit)));
         }
     }
+
+    @Override
+    public boolean equals(Object other) {
+        if (other == this) {
+            return true;
```

Comment:
Looks like this method is not tested, maybe we can add some tests for here 👍

Reply:
Thanks for pointing it out! Will add tests to cover it